tests/cupyx_tests/scipy_tests/sparse_tests/test_index.py

```diff
@@ -0,0 +1,110 @@
+import unittest
+
+import cupy
+from cupy import sparse
+from cupy import testing
+
+import numpy
+
+import pytest
+
+
+@testing.parameterize(*testing.product({
+    'format': ['csr', 'csc'],
+    'density': [0.1, 0.4, 0.9],
+    'dtype': ['float32', 'float64', 'complex64', 'complex128'],
+    'n_rows': [25, 150],
+    'n_cols': [25, 150]
+}))
+@testing.with_requires('scipy')
+@testing.gpu
+class TestIndexing(unittest.TestCase):
+
+    def _run(self, maj, min=None):
+        a = sparse.random(self.n_rows, self.n_cols,
+                          format=self.format,
+                          density=self.density)
+
+        # sparse.random doesn't support complex types
+        # so we need to cast
+        a = a.astype(self.dtype)
+
+        maj_h = maj.get() if isinstance(maj, cupy.ndarray) else maj
+        min_h = min.get() if isinstance(min, cupy.ndarray) else min
```

Comment:
This is not needed. I think it's fine for the indices to stay on GPU.

Reply:
This is most certainly not needed until the fancy indexing PRs.

Reply:
Ah OK. I think I got myself too confused today by going back and forth the big PR and the break-out ones...